Console/UpgradeHelperCommand.php

```diff
@@ -32,12 +44,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $diff = file($input->getArgument('diff'));
         $result = $this->runner->run($diff);
 
+        $fileDirectoryPath = $this->directoryList->getPath(DirectoryList::ROOT);
+        $fileName = 'UpgradeHelperResult_'.date('m-d-Y').'.csv';
+        $filePath = $fileDirectoryPath . '/' . $fileName;
+        $data['headers'] = ['Patched', 'Customized', 'Reviewed (Yes / No)', 'Notes'];
+
         foreach ($result as $type => $items) {
             $output->writeln('-------- ' . $type . ' --------');
             foreach ($items as $patched => $customized) {
+                $data[] = [$patched, $customized];
                 $output->writeln('Patched: ' . $patched);
                 $output->writeln('Customized: ' . $customized);
             }
         }
+        $this->csvProcessor
```

Comment:
I think we probably want something like an output flag to determine if it should be output as CSV or printed to the terminal.